### PR TITLE
Add a Rerun button

### PR DESCRIPTION
**Pull request by jlgarciasanchez**

### CODE REVIEW

File: src/main/webapp/js/app/routers/mobileRouter.js [hunk 1]
```diff
@@ -212,14 +214,87 @@ define([
...
+        		testObject = tasks[0].testObject;
+        		argumentList = tasks[0].ArgumentList;
+        		_.each( tasks, function( task ) {
+                    if(!_.isUndefined(task) && !_.isUndefined(task.executableTestSuite) && !_.isUndefined(task.executableTestSuite.id)) {
+                    	etslocal[task.executableTestSuite.id] = collection._byId[task.executableTestSuite.id];
+                    	etsremote.push(task.executableTestSuite.id);
+            		}
+    			});
+        	}
+			if(testObject.hasOwnProperty("remoteResource")){
```
[the remark this answers is not in the document]
Comment (carlospzurita) [reply src/main/webapp/js/app/routers/mobileRouter.js]: Hi Jon,

We tried to reproduce this error following these steps, but could not get it. 

¿Could you please share some more information about your testing: browser, OS, tests conformance classes...?

Thank you.

File: src/main/webapp/js/app/routers/mobileRouter.js [hunk 2]
```diff
@@ -212,14 +214,87 @@ define([
...
+				//Rerun a remote file test automatically
+				
+				//Generate a new label.
+	        	var regex1 = RegExp(/\(\d+\)$/);
+	        	var regex2 = RegExp(/\d+/);
+	        	if(regex1.test(label)){
+	        		var cadenaIntentos = regex1.exec(label);
```
[the remark this answers is not in the document]
Comment (carlospzurita) [reply src/main/webapp/js/app/routers/mobileRouter.js]: Fixed in https://github.com/guadaltel/etf-webapp/commit/51881ee9b868140bb590e6c84a63d8b0f79dd59c

File: src/main/webapp/layout.html [hunk 3]
```diff
@@ -58,7 +58,7 @@ <h1 th:if="${T(de.interactive_instruments.etf.webapp.helpers.View).brandingText
 <!--/* Footer */-->
 <div tiles:fragment="footer">
 <div data-role="footer" data-position="fixed">
-    <h1>ETF WebApp <span th:text="${T(de.interactive_instruments.etf.webapp.helpers.View).version}">X.Y.Z</span> &copy; 2017-2018 European Union, interactive instruments GmbH</h1>
+    <h1>ETF WebApp <span th:text="${T(de.interactive_instruments.etf.webapp.helpers.View).version}">X.Y.Z</span> &copy; 2017-2018 European Union</h1>
```
[the remark this answers is not in the document]
Comment (carlospzurita) [reply src/main/webapp/layout.html]: Fixed in https://github.com/guadaltel/etf-webapp/commit/51881ee9b868140bb590e6c84a63d8b0f79dd59c

File: src/main/webapp/js/app/routers/mobileRouter.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (jonherrmann) [reply src/main/webapp/js/app/routers/mobileRouter.js]: Hi Carlos,

I tried to reproduce it once more and realized that it was related [to a problem with a CDN](https://github.com/etf-validator/etf-webapp/issues/154). The issue only occurs if I use HTTP without S.  Thanks, I would not have noticed the other issue. No change required, will mark this as resolved.

**Pull request closed without merging**